community/values/src/main/java/org/neo4j/values/storable/DurationValue.java

```diff
@@ -85,7 +85,7 @@ public static DurationValue duration( Period value )
 
     public static DurationValue duration( long months, long days, long seconds, long nanos )
     {
-        return newDuration( months, days, seconds, (int) nanos );
+        return newDuration( months, days, seconds, nanos );
```

Comment:
What happened to normalization of nanos? The previous state looks broken, shouldn't this be fixed in 3.4.0 ?

Reply:
The previous state _is_ broken. Normalization of nanos happens and introduced this bug. The general decision was to delay this until 3.4.1 if no other worse release blocker appeared.